syncd/Syncd.cpp

```diff
@@ -4295,6 +4296,14 @@ bool Syncd::isVeryFirstRun()
     return firstRun;
 }
 
+static void timerWatchdogCallback(
+        _In_ int64_t span)
+{
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_ERROR("main loop execution exceeded %ld ms", span);
```

Comment:
This is in millisecond (ms) or microsecond (us)

Reply:
microseconds :P

Reply:
fixed